packages/core/src/cli/validate.ts

```diff
@@ -76,8 +85,13 @@ export function getFunctionArgs(parsedArgs: minimist.ParsedArgs, extraArgs: stri
     throw new Error('The validate command takes only one argument: the build info directory.');
   } else {
     const buildInfoDir = extraArgs.length === 1 ? undefined : extraArgs[1];
+    const contract = parsedArgs['contract'];
+    const reference = parsedArgs['reference'];
+    if (reference !== undefined && contract === undefined) {
```

Comment:
It's possible to pass `--contract --reference Foo` and it results in `contract: '', reference: 'Foo'`. This sounds like something we would want to catch in this function.

Just `--contract --reference` is also probably valid.